### PR TITLE
Added support to parent_id query

### PR DESCRIPTION
**Pull request by giovannialbero1992**
added ParentId Query by reference [#1287](https://github.com/ruflin/Elastica/issues/1287)

### CODE REVIEW

File: lib/Elastica/Query/ParentId.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Elastica\Query;
+
+final class ParentId extends AbstractQuery
```
Comment (ruflin) [inline lib/Elastica/Query/ParentId.php]: Why did you add final here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ All notable changes to this project will be documented in this file based on the
 ### Added
 
  - Parameter `filter_path` for response filtering (e.g. `$index->search($query, ['filter_path' => 'hits.hits._source'])`)
-
+ - Added `\Elastica\Query\ParentId` to avoid join with parent documents [#1287](https://github.com/ruflin/Elastica/issues/1287)
```
Comment (ruflin) [inline CHANGELOG.md]: Could also add here a newline before the next title?

File: lib/Elastica/Query/ParentId.php (hunk 1 above)
Comment (giovannialbero1992) [inline lib/Elastica/Query/ParentId.php]: I usually apply `final` by default to classes that are not specifically designed to be extended. 
This avoids breaking an eventual child class when refactoring this class.
If you want i can remove `final` from this class.

Comment (ruflin) [inline lib/Elastica/Query/ParentId.php]: I would prefer to remove it as we don't have it in any other files and I don't see a reason, why we should prevent someone extendings this class if we wants to (even though it is unlikely to happen).

Comment (giovannialbero1992) [inline lib/Elastica/Query/ParentId.php]: Ok, I've removed final

**Comment by ruflin** [comment]
@giovannialbero1992 Seems like you have changelog conflict :-(

**Comment by giovannialbero1992** [comment]
@ruflin done! ;-)

**Comment by ruflin** [comment]
@giovannialbero1992 Thanks. merged.
